catalog.html

```diff
@@ -0,0 +1,231 @@
+          <span>Перфоратор BOSCH BFG 6000</span>
+          <span>30500 Р.</span>
+          <a href="#">25500 Р.</a>
+        </li>
+        <li><a href="#">КУПИТЬ</a>
+          <a href="#">В ЗАКЛАДКУ</a>
+          <img src="" alt="">
+          <span>Перфоратор BOSCH BFG 2000</span>
+          <a href="#">12500 Р.</a>
+        </li>
+        <li><a href="#">КУПИТЬ</a>
+          <a href="#">В ЗАКЛАДКУ</a>
+          <img src="" alt="">
+          <span>Перфоратор BOSCH BFG 3000</span>
+          <span>22500 Р.</span>
+          <a href="#">15500 Р.</a>
+        </li>
+        <li><a href="#">КУПИТЬ</a>
+          <a href="#">В ЗАКЛАДКУ</a>
+          <img src="" alt="">
+          <span>Перфоратор BOSCH BFG 6000</span>
+          <span>30500 Р.</span>
+          <a href="#">25500 Р.</a>
+        </li>
+        <li><a href="#">КУПИТЬ</a>
+          <a href="#">В ЗАКЛАДКУ</a>
+          <img src="" alt="">
+          <span>Перфоратор BOSCH BFG 2000</span>
+          <a href="#">25500 Р.</a>
+        </li>
+      </ul>
+    </section>
+    <section>
+      <ul>
+        <li><a href="#">1</a></li>
+        <li><a href="#">2</a></li>
+        <li><a href="#">3</a></li>
+        <li><a href="#">СЛЕДУЮЩАЯ</a></li>
+      </ul>
+    </section>
+    <section>
+      <h2>ПАРА СЛОВ О ПЕРФОРАТОРАХ</h2>
+      <p>Перфоратор — универсальный строительный инструмент. Предназначен для сверления и долбления отверстий в кирпиче, бетоне и камне.<br>
+        Существуют электромеханические и пневматические перфораторы. Современные модели могут выполнять функции дрели, шуруповерта и отбойного молотка.<br>
+        В ассортиментном ряду есть выбор «трехрежимников»: в одном устройстве будут совмещены отбойный молоток и ударная дрель.</p>
+    </section>
+
+  </main>
+  <footer>
+    <section>
+
+      <h2>
+        <a href="#">Техномарт</a>
+      </h2>
+      <p>
+        г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8<br>
+        +7 (812) 555-05-55
+      </p>
+    </section>
+    <section>
+
+      <ul>
+        <li><a href="#">Компания</a></li>
+        <li><a href="#">Новости</a></li>
+        <li><a href="#">Каталог</a></li>
+        <li><a href="#">Доставка</a></li>
+        <li><a href="#">Контакты</a></li>
+      </ul>
+    </section>
+    <section>
+      <ul>
+        <li><a href="#">Материал</a></li>
+        <li><a href="#">Техника</a></li>
+        <li><a href="#">Инструмент</a></li>
+        <li><a href="#">Спецпредлжения</a></li>
+      </ul>
+    </section>
+    <section>
+      <p>
+        © 2010-2016 Компания «Техномарт»<br>
+        Все права защищены
+      </p>
+    </section>
+    <section>
+      <ul>
+        <li>
+          <a href="#">Вконтакте</a>
+        </li>
+        <li>
+          <a href="#">Фейсбук</a>
+        </li>
+        <li>
+          <a href="#">Инстаграм</a>
+        </li>
+      </ul>
+    </section>
+    <section>
+      <p>Обратная связь:</p><br>
+      <a href="mail@htmlacademy.ru">mail@htmlacademy.ru</a>
```

Comment:
```suggestion
      <a href="mailto:mail@htmlacademy.ru">mail@htmlacademy.ru</a>
```
Если добавить `mailto:` к ссылке, то откроется встроенный в компьютер почтовый сервис. Это очень удобно. Старайся всегда добавлять `mailto:` к адресам, на который задумано писать письма